src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -4000,6 +4002,38 @@ def test_set_up_storage_profile(self):
         ground_truth_mc_1 = self.models.ManagedCluster(location="test_location", storage_profile=storage_profile_1)
         self.assertEqual(dec_mc_1, ground_truth_mc_1)
 
+    def test_set_up_custom_ca_trust_certificates(self):
+        dec_1 = AKSPreviewManagedClusterCreateDecorator(
+            self.cmd,
+            self.client,
+            {"custom_ca_trust_certificates": get_test_data_file_path("certs.txt")},
+            CUSTOM_MGMT_AKS_PREVIEW,
+        )
+        mc_1 = self.models.ManagedCluster(location="test_location")
+        dec_1.context.attach_mc(mc_1)
+        dec_mc_1 = dec_1.set_up_custom_ca_trust_certificates(mc_1)
+        test_cert_string = '-----BEGIN CERTIFICATE-----\n' \
```

Comment:
I found that the `test_cert_string` used in all tests seems to be the same. Could you define it as a constant and reuse it in the multiple tests?

Reply:
done